config/ssl/localhost.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think this would be better as a rake task to generate an SSL cert specific to the user, cheers